packages/foreman/foreman/foreman.provreq

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python2
+
+"""
+Automatic provides and requires generator for webpack bundles
+"""
+
+import json
+import os
+import sys
+
+FOREMAN_MANIFEST = '/var/lib/foreman/public/webpack/manifest.json'
+
+paths = [path.rstrip() for path in sys.stdin.readlines()]
```

Comment:
Maybe we should move these definitions to below the function definitions?

Reply:
we should